yandexcloud/_channels.py

```diff
@@ -13,10 +17,16 @@
     VERSION = "0.0.0"
 
 SDK_USER_AGENT = "yandex-cloud-python-sdk/{version}".format(version=VERSION)
+logger = logging.getLogger(__name__)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
I believe it is very bad pattern to setup logging without user's permission.
Now, if I want to setup logging on my own, I need to remove this handler somehow.

I saw in several libraries logging setup in case there are some unique variable in env, like YC_LOG_LEVEL.

Reply:
fixed, ye, it was test code